### PR TITLE
[helm][tf] backport alertmanager fixes to 1.3

### LINKED ISSUE #4630: [nms] Alert receivers reported not working
### Your Environment

- **Version: master
- **Affected Component:** NMS
- **Affected Subcomponent:** Alerts Receivers
- **Deployment Environment:** e.g. AWS

### Describe the Issue

Set up alerts receivers are reported not working. Requires additional investigation.

**Expected behavior**
N/A

**Screenshots**
N/A

**Additional context**
N/A

### PR DESCRIPTION
**Pull request by Scott8440**
Signed-off-by: Scott8440 <scott8440@gmail.com>

## Summary
Updates v1.3 terraform modules to deploy latest versions of prometheus-configmanagers and corrects orchestrator.yml config generation.

## Test Plan
Deploy following 1.3 terraform instructions.

### CODE REVIEW

**Review by karthiksubraveti — APPROVED** [review]

File: orc8r/cloud/deploy/terraform/orc8r-helm-aws/variables.tf [hunk 1]
```diff
@@ -265,3 +265,19 @@ variable "deploy_openvpn" {
...
+  description = "Image version for prometheus configurer."
+  type        = string
+  default     = "1.0.4"
+}
+
+variable "alertmanager_configurer_version" {
+  description = "Image version for alertmanager configurer."
+  type        = string
+  default     = "1.0.4"
+}
```
Comment (119Vik) [inline orc8r/cloud/deploy/terraform/orc8r-helm-aws/variables.tf]: can you please add newline in the end

**Comment by 119Vik** [comment]
@Scott8440 I can confirm that this change is fixing the issue.

**Review by 119Vik — APPROVED** [review]
This change is fixing the problem. 
Can You please add a newline in the end of the file.

**Comment by ekowtaylor** [comment]
@karthiksubraveti, @Scott8440 is this change still relevant is this ready to be merged?

**Review by mpgermano — APPROVED** [review]

**Comment by Scott8440** [comment]
no idea how to fix feg build failure :)

**Review by hcgatewood — APPROVED** [review]
Messaged @Scott8440 about this on workchat. I'm assuming this is still relevant for backporting, but I'll defer to him

**Comment by themarwhal** [comment]
Overriding the required check for lte-test since it is not running
Will look into why

**Pull request merged**